app/src/pages/confirmation.tsx

```diff
@@ -1,20 +1,34 @@
-import { useAppContext } from '@context/state'
```

Comment:
Notes for reviewers: The Confirmation page (`/confirmation`) is the second of two "clearable" pages, pages where the user can click a button and their session storage will be manually cleared (both the session storage and the `session` state). 

This page has no back link and the call to action button ("start a new application") routes the user to `/`. There are no data validation guards for this page because it is a page intended to facilitate deleting data.

![Screen Shot 2022-10-28 at 2 58 11 AM](https://user-images.githubusercontent.com/67701/198560569-75cea09f-8004-4a5d-9da4-eb13e8df3763.png)